src/commands.ts

```diff
@@ -598,7 +598,7 @@ export const run = async (functionName: string, cmd: { nondev: boolean }) => {
   async function runFunction(functionName: string) {
     try {
       // Load local credentials.
-      await loadAPICredentials(true);
+      await loadAPICredentials();
```

Comment:
Logically speaking, we cannot use credentials that are global.
We must force local.